apps/judicial-system/web/src/routes/Judge/Ruling/StepOne/RulingStepOne.tsx

```diff
@@ -132,7 +135,7 @@ export const RulingStepOne: React.FC = () => {
     }
 
     if (workingCase) {
-      setIsStepIllegal(isNextDisabled(requiredFields))
+      setIsStepIllegal(isNextDisabled(requiredFields) || !workingCase.decision)
```

Comment:
Can radio buttons not be part of required fields?

Reply:
That would require us to refactor the validation. The validation uses regex to validate values and in this case we are checking if `workingCase.decision` is `null` and I don't know how to write a regex that checks for nulls.